src/plugins/data/public/antlr/opensearch_ppl/code_completion.ts

```diff
@@ -40,16 +40,50 @@ export const getSuggestions = async ({
       finalSuggestions.push(...fetchFieldSuggestions(indexPattern, (f: any) => `${f} `));
     }
 
+    if (suggestions.suggestValuesForColumn) {
+      // get dataset for connecting to the cluster currently engaged
+      const dataset = services.data.query.queryString.getQuery().dataset;
+
+      // take the column and push in values for that column
+      const res = await fetchColumnValues(
+        [indexPattern.title],
+        suggestions.suggestValuesForColumn,
+        services,
+        dataset
+      );
+
+      let i = 0;
+      finalSuggestions.push(
+        ...res.body.fields[0].values.map((val: any) => {
+          i++;
+          return {
+            text: val.toString(),
+            insertText: typeof val === 'string' ? `"${val}" ` : `${val} `,
+            type: monaco.languages.CompletionItemKind.Value,
+            detail: SuggestionItemDetailsTags.Value,
+            sortText: i.toString().padStart(3, '0'),
```

Comment:
seems it's sorting by insertion order, is sortText with the 0 padding required? why not sort alphabetically?